pkg/inspecttypes/native/volume.go

```diff
@@ -21,4 +21,5 @@ type Volume struct {
 	Name       string             `json:"Name"`
 	Mountpoint string             `json:"Mountpoint"`
 	Labels     *map[string]string `json:"Labels,omitempty"`
+	Size       string             `json:"Size,omitempty"`
```

Comment:
Shouldn't this be int64?

Reply:
Human-readable string can be printed in `nerdctl volume ls --size`

Reply:
I am converting to human readable form. I can change to int64

Reply:
Will make the change! Thanks